src/stores/user-icon-collection.store.ts

```
@@ -5,17 +5,16 @@ import { UserIconCollectionDBService } from "../services/user-icon-collection-in
-    UserIconCollectionDBService.getList().then(set).then(selectedCollection.selectDefault);
-    // Subscribe to changes in the IndexedDB service
-    const unsubscribe = UserIconCollectionDBService.subscribe(set);
+	const { subscribe, set } = writable<UserIconCollection[]>([]);
 
-    return {
-        subscribe,
-        set
-    };
+	UserIconCollectionDBService.getList().then(set).then(selectedCollection.selectDefault);
+	// Subscribe to changes in the IndexedDB service
+	const unsubscribe = UserIconCollectionDBService.subscribe(set);
```

ESLint

[error] 'unsubscribe' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)